### DISCUSSION TITLE
Element Verification To Assist w/ Identifying If Current Xpath is Still Valid or Assist w/ Renaming in Element Library

### DISCUSSION THREAD

**Discussion by Laos-Champion** [testsigmahq/testsigma#177] [Feature Ideas]
A feature that I am hoping to see, and that TestSigma's competitors' have, is the ability to Identify elements within the recording window.  This makes it easier to identify and rename elements within the library.  It also helps in determining whether the currently assigned xpath value is still valid or not.

In the provided screenshot, when an element is selected, and the user clicks on the magnifying glass, the element blinks green on the page along with a match found toast.  When there are multiple matches, all matching elements blink orange and when there are no matches, the entire screen blinks red.  I'm not saying that it needs to be this way but something along these lines would be helpful and greatly appreciated.
<img width="2537" alt="Screen Shot 2022-09-23 at 11 58 38 AM" src="https://user-images.githubusercontent.com/110565278/192038916-23680cfd-ce41-4872-b137-1208bab3d5f4.png">

<img width="2510" alt="Screen Shot 2022-09-23 at 2 10 51 PM" src="https://user-images.githubusercontent.com/110565278/192041051-02d50ecd-c421-42e6-b5f1-d5031b3520ad.png">

**Comment by amitsavyasachi** [comment]
@Laos-Champion This is a valid request. We are taking this up with our internal Product discussion. We will share an update here
